app/javascript/turbo/cable.ts

```diff
@@ -0,0 +1,26 @@
+import type {
+  Consumer,
+  Mixin,
+  createConsumer as ActionCableCreateConsumer,
+  ChannelNameWithParams
+} from '@rails/actioncable'
+
+let consumer: Consumer | undefined
+
+export async function getConsumer() {
+  return consumer || createConsumer().then(setConsumer)
+}
+
+export function setConsumer(newConsumer: Consumer) {
+  return consumer = newConsumer
+}
+
+export async function createConsumer() {
+  const { createConsumer } = await import(/* webpackChunkName: "actioncable" */ "@rails/actioncable/src")
```

Comment:
I could't set `"strict": true` in tsconfig.json because I couldn't resolve the type error on this line.

Reply:
I have fixed the type error.